schemas/telemetry/configuration-schema.json

```diff
@@ -101,7 +103,8 @@
                 },
                 "default_privacy_level": {
                   "type": "string",
-                  "description": "Session replay default privacy level"
+                  "description": "Session replay default privacy level",
+                  "readOnly": false
```

Comment:
Same question.

Reply:
The thinking was that Flutter's implementation of Session Replay will likely be completely independent of native session replay. I won't be sending down the sampling rate or privacy level to the native configuration objects, so I'll need to supply the values from the Flutter plugin.